docs/edge/enable-edge.md

```diff
@@ -48,13 +48,6 @@ export EDGES_RPC_PORT=**NEW_EDGES_RPC_PORT**
 
 {% include templates/edge/ssl-grpc-note.md %}
 
-#### Restart the service
-
-```bash
-$ sudo service thingsboard stop
-$ sudo service thingsboard start
-```
-
```

Comment:
why did you remove these lines?
Once config is updated service must be restarted.

Reply:
sorry, fixed this